Atm error

### What steps will reproduce the issue?  
1. Use atm in console
### What was supposed to happen?
Atm open in console
### What happened instead? 
Error
### What version of TimeIsMoney are you using?  
1.9.6.2
### What Spigot version are you using? Paste the output of /version below.  
1.12
### What plugins do you have installed? Paste the output of /plugins below.  
Vault, iConomy, Essentials, ProtocolLib, TimeIsMoney
### Are any errors related to TimeIsMoney in your console or logs? If so, paste below.  
```
[12:40:22 WARN]: Unexpected exception while parsing console command "atm"
org.bukkit.command.CommandException: Unhandled exception executing command 'atm' in plugin TimeIsMoney v1.9.6.2
  at org.bukkit.command.PluginCommand.execute(PluginCommand.java:46) ~[spigot-1.12.jar:git-Spigot-596221b-9a1fc1e]
  at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:141) ~[spigot-1.12.jar:git-Spigot-596221b-9a1fc1e]
  at org.bukkit.craftbukkit.v1_12_R1.CraftServer.dispatchCommand(CraftServer.java:641) ~[spigot-1.12.jar:git-Spigot-596221b-9a1fc1e]
  at org.bukkit.craftbukkit.v1_12_R1.CraftServer.dispatchServerCommand(CraftServer.java:627) [spigot-1.12.jar:git-Spigot-596221b-9a1fc1e]
  at net.minecraft.server.v1_12_R1.DedicatedServer.aP(DedicatedServer.java:443) [spigot-1.12.jar:git-Spigot-596221b-9a1fc1e]
  at net.minecraft.server.v1_12_R1.DedicatedServer.D(DedicatedServer.java:406) [spigot-1.12.jar:git-Spigot-596221b-9a1fc1e]
  at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:678) [spigot-1.12.jar:git-Spigot-596221b-9a1fc1e]
  at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576) [spigot-1.12.jar:git-Spigot-596221b-9a1fc1e]
  at java.lang.Thread.run(Thread.java:745) [?:1.8.0_112]
Caused by: java.lang.ClassCastException: org.bukkit.craftbukkit.v1_12_R1.command.ColouredConsoleSender cannot be cast to org.bukkit.entity.Player
  at de.Linus122.TimeIsMoney.ATM.onCommand(ATM.java:383) ~[?:?]
  at org.bukkit.command.PluginCommand.execute(PluginCommand.java:44) ~[spigot-1.12.jar:git-Spigot-596221b-9a1fc1e]
  ... 8 more
```
### Any additional information that you would like to provide that may be relevant to the issue?
No

--- a/Plugin/src/main/java/de/Linus122/TimeIsMoney/ATM.java
+++ b/Plugin/src/main/java/de/Linus122/TimeIsMoney/ATM.java
@@ -378,7 +378,7 @@ public class ATM implements Listener, CommandExecutor {
 	
 	@Override
 	public boolean onCommand(CommandSender cs, Command arg1, String arg2, String[] args) {
-		if (!(cs instanceof Player)){
+		if (!(cs instanceof Player)) {
 			cs.sendMessage("Only players can use atms.");
 			return true;
 		}

--- a/Plugin/src/main/java/de/Linus122/TimeIsMoney/ATM.java
+++ b/Plugin/src/main/java/de/Linus122/TimeIsMoney/ATM.java
@@ -378,6 +378,10 @@ public class ATM implements Listener, CommandExecutor {
 	
 	@Override
 	public boolean onCommand(CommandSender cs, Command arg1, String arg2, String[] args) {
+		if (!(cs instanceof Player)){
+			cs.sendMessage("Only players can use atms.");
+			return true;
+		}
 		if (args.length == 0) {
 			if (cs.hasPermission("tim.use")) {
 				openGUI((Player) cs);

